File: Samples/Layout/HelloWorld/VB.NET/HelloWorld2.vb
Imports BitMiracle.Docotic.Pdf.Layout

Namespace BitMiracle.Docotic.Pdf.Samples
    Class HelloWorld2
        Public Shared Sub CreatePdf()
            PdfDocumentBuilder.Create().Generate("hello.pdf", AddressOf BuildDocument)
        End Sub

        Private Shared Sub BuildDocument(doc As Document)
            doc.Pages(AddressOf BuildPages)
        End Sub

        Private Shared Sub BuildPages(pages As PageLayout)
            pages.Content().Text("Hello, world!")
        End Sub
    End Class
End Namespace

File: Samples/Layout/HelloWorld/C#/HelloWorld2.cs
using BitMiracle.Docotic.Pdf.Layout;

namespace BitMiracle.Docotic.Pdf.Samples
{
    class HelloWorld2
    {
        public static void CreatePdf()
        {
            PdfDocumentBuilder.Create().Generate("hello.pdf", BuildDocument);
        }

        static void BuildDocument(Document doc)
        {
            doc.Pages(BuildPages);
        }

        static void BuildPages(PageLayout pages)
        {
            pages.Content().Text("Hello, world!");
        }
    }
}
